sw/device/lib/usbdev.c

```diff
@@ -300,3 +300,43 @@ void usbdev_init(usbdev_ctx_t *ctx, bool pinflip, bool diff_rx, bool diff_tx) {
   REG32(USBDEV_BASE_ADDR + USBDEV_USBCTRL_REG_OFFSET) =
       (1 << USBDEV_USBCTRL_ENABLE_BIT);
 }
+
+void usbdev_force_dx_pullup(line_sel_t line, bool set) {
+  // Force usb to pretend it is in suspend
+  uint32_t reg_val = REG32(USBDEV_BASE_ADDR + USBDEV_PHY_PINS_DRIVE_REG_OFFSET);
+  uint32_t mask;
+
+  if (line == kDpSel) {
+    mask = USBDEV_PHY_PINS_DRIVE_DP_PULLUP_EN_O_BIT;
+  } else {
+    mask = USBDEV_PHY_PINS_DRIVE_DN_PULLUP_EN_O_BIT;
+  }
```

Comment:
These names are really really similar... I thought this if statement could be folded away.

Reply:
sorry i didn't completely follow. 
Do you mean you want me to use a tenerary statement or something?

Reply:
let me know if this is what you meant.